package.json

```diff
@@ -50,7 +50,8 @@
     "README.md",
     "third-party-podspecs",
     "template",
-    "local-cli"
+    "local-cli",
+    "template.config.js"
```

Comment:
can you confirm with `npm pack` that `.gitignore` from `template/`is published as well?